### PR TITLE
Styling set up

### PR DESCRIPTION
**Pull request by mbouzi**
- basic structure set up for styling

mocks - https://www.figma.com/file/g7doUVwCahjA5YHJNQnNd7A5/Tardigrade.io?node-id=1111%3A0

yarn install & yarn start to get server started at localhost:3000

### CODE REVIEW

File: site/assets/sass/main.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* Base CSS files */
+@import "color";
+@import "fonts";
+@import "base";
+
+/* Pages */
+// @import "home";
```
Comment (VitaliiShpital) [inline site/assets/sass/main.scss]: do we need this comment?

File: site/layouts/partials/banner.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{{ with .Site.GetPage "banner/banner.md"}}
+<div class="container-fluid banner-bg">
+    <div class="container">
+        <div class="row">
+            <div class="col-xs-7 col-sm-6 banner-column">
+                <p class="banner-left">{{.Params.leftText}}</p>
+            </div>
+            <div class="col-xs-5 col-sm-6 banner-column text-right">
+                <a class="banner-cta" href="https://tardigrade.io">
```
Comment (VitaliiShpital) [inline site/layouts/partials/banner.html]: should we use target="_blank" here?

File: site/layouts/partials/banner.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+                    <p class="banner-right">{{.Params.rightText}}</p>
+                    <img src="/img/tardigrade-logo-only.svg" height="10" width="17"/><span class="glyphicon glyphicon-chevron-right"></span>
```
Comment (VitaliiShpital) [inline site/layouts/partials/banner.html]: should we add alt="tardigrade logo" to img tag?

make span tag start from a new line?

**Comment by mbouzi** [comment]
made all changes based on feedback, since they are small I will merge

**Pull request merged**